scss/_base_forms.scss

```diff
@@ -21,11 +21,12 @@
     -moz-appearance: textfield;
     appearance: textfield;
     // stylelint-enable property-no-vendor-prefix
-    background-color: $colors--light-theme--background-inputs;
+    background-color: $colors--theme--background-inputs;
     border: 0 solid transparent;
-    border-bottom: $input-border-thickness solid $colors--light-theme--text-default;
+    border-bottom: $input-border-thickness solid $colors--theme--text-default;
```

Comment:
I think on standup we talked about making it use the high contrast border color instead?

Reply:
Oh, it's defined as high contrast border on the next line.

I guess we don't need the next line then? Just the `border-bottom:` definition using the correct colour should be fine? Defining border colour in 2 places makes it confusing.